common-util/src/test/java/org/sunbird/common/models/util/ProjectUtilTest.java

```diff
@@ -15,19 +15,18 @@
 import java.util.TimeZone;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.velocity.VelocityContext;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.sunbird.common.exception.ProjectCommonException;
 import org.sunbird.common.request.Request;
 import org.sunbird.common.responsecode.ResponseCode;
 
 /** Created by arvind on 6/10/17. */
-public class ProjectUtilTest {
+public class ProjectUtilTest extends BaseForHttpTest {
 
   private PropertiesCache propertiesCache = ProjectUtil.propertiesCache;
 
   @Test
-  public void testMailTemplateContextNameAsent() {
+  public void testMailTemplateFailureWithNameAbsent() {
```

Comment:
Check naming of tests. e.g. testMailTemplateFailureWithNameAbsent to testGetContextFailureWithoutName